recipes-openxt/initrdscripts/initramfs-xenclient/init.sh

```diff
@@ -169,10 +169,10 @@ tpm_setup() {
     insmod /lib/modules/$(uname -r)/kernel/drivers/char/tpm/tpm.ko
     insmod /lib/modules/$(uname -r)/kernel/drivers/char/tpm/tpm_tis.ko
     mknod /dev/tpm0 c 10 224
-    echo -n "initramfs measuring $ROOT_DEVICE: "
```

Comment:
Never understood the case and really don't like that it was being done. If that is in fact the reason, it should be dropped all together. do you disagree?